deployment/ansible/filter_plugins/custom_filters.py

```diff
@@ -0,0 +1,45 @@
+class FilterModule(object):
+    ''' Additional filters for use within Ansible. '''
+
+    def filters(self):
+        return {
+            'is_not_in': self.is_not_in,
+            'is_in': self.is_in,
+            'some_are_in': self.some_are_in
+        }
+
+    def is_not_in(self, *t):
+        """Determnies if there are no elements in common between x and y
+
+            x | is_not_in(y)
+
+        Arguments
+        :param t: A tuple with two elements (x and y)
+        """
+        groups_to_test, all_group_names = t
```

Comment:
[Core plugins](https://github.com/ansible/ansible/blob/devel/lib/ansible/runner/filter_plugins/core.py) had a similar method signature. Looking at them again, these might be better fixed to two elements.